docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
       PACKET_ENV: testing
       PACKET_VERSION: 42
       ROLLBAR_TOKEN: 42
+    ports:
+      - 42111
```

Comment:
While I'm sure this was helpful for testing, I'm not certain it should be checked in.

Reply:
> While I'm sure this was helpful for testing, I'm not certain it should be checked in.

I dont know how grpc health check can be triggered with out ports.

Reply:
Well, the docker-compose file includes a client container that is linked to the server container. In theory the client could be upgraded to be able to do the healthcheck and then you could do the check from inside the client container.